src/main/java/seedu/address/model/person/Person.java

```diff
@@ -191,8 +191,12 @@ public String toString() {
     @Override
     public int compareTo(Person another) {
         int nameCompare = this.name.fullName.compareTo(another.name.fullName);
-        int companyCompare = this.company.company.compareTo(another.company.company);
-        int jobCompare = this.company.company.compareTo(another.jobTitle.jobTitle);
+        int companyCompare = (this.company != null && another.company != null)
+                ? this.company.company.compareTo(another.company.company)
```

Comment:
Should we change the company variable in the company class to another name to make it less confusing?

Reply:
I think it was `companyName` previously, but was changed throughout for brevity. Changing the company variable in the company class will require a lot of changes throughout the code base. I think it will be better if I rename `this.company` to something else to avoid confusion.